debian/smf-spf.postinst

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 DATADIR=/var/run/smfs
-CONFDIR=/etc/mail/smfs
+CONFDIR=/etc
```

Comment:
Why change this to /etc?

Reply:
This package needs a single config file. And also we have to create the mail/smfs directory